R/make_units.R

```diff
@@ -440,3 +440,12 @@ drop_units.units <- function(x) {
   x
 }
 
+#' @export
+drop_units.data.frame <- function(.data){
```

Comment:
An S3 method should be consistent with its generic: in this case, the argument must be `x` instead of `.data`.